src/test/java/seedu/address/logic/commands/assignment/AddAssignmentCommandTest.java

```diff
@@ -0,0 +1,236 @@
+        AddAssignmentCommand addOneCommand = new AddAssignmentCommand(assignment1);
+        AddAssignmentCommand addTwoCommand = new AddAssignmentCommand(assignment2);
+
+        // same object -> returns true
+        assertTrue(addOneCommand.equals(addOneCommand));
+
+        // same values -> returns true
+        AddAssignmentCommand addOneCommandCopy = new AddAssignmentCommand(assignment1);
+        assertTrue(addOneCommand.equals(addOneCommandCopy));
+
+        // different types -> returns false
+        assertFalse(addOneCommand.equals(1));
+
+        // null -> returns false
+        assertFalse(addOneCommand.equals(null));
+
+        // different person -> returns false
+        assertFalse(addOneCommand.equals(addTwoCommand));
+    }
+
+    @Test
+    public void toStringMethod() {
+        AddAssignmentCommand addCommand = new AddAssignmentCommand(ASSIGNMENT1);
+        String expected = AddAssignmentCommand.class.getCanonicalName() + "{toAdd=" + ASSIGNMENT1 + "}";
+        assertEquals(expected, addCommand.toString());
+    }
+
+    /**
+     * A default model stub that have all of the methods failing.
+     */
+    private class ModelStub implements Model {
+        @Override
+        public void setUserPrefs(ReadOnlyUserPrefs userPrefs) {
+            throw new AssertionError("This method should not be called.");
+        }
+
+        @Override
+        public ReadOnlyUserPrefs getUserPrefs() {
+            throw new AssertionError("This method should not be called.");
+        }
+
+        @Override
+        public GuiSettings getGuiSettings() {
+            throw new AssertionError("This method should not be called.");
+        }
+
+        @Override
+        public void setGuiSettings(GuiSettings guiSettings) {
+            throw new AssertionError("This method should not be called.");
+        }
+
+        @Override
+        public Path getAddressBookFilePath() {
+            throw new AssertionError("This method should not be called.");
+        }
+
+        @Override
+        public void setAddressBookFilePath(Path addressBookFilePath) {
+            throw new AssertionError("This method should not be called.");
+        }
+
+        @Override
+        public void addPerson(Person person) {
+            throw new AssertionError("This method should not be called.");
+        }
+
+        @Override public void addAssignment(Assignment assignment) {
+            throw new AssertionError("This method should not be called.");
+        }
+
+        @Override
+        public void setAddressBook(ReadOnlyAddressBook newData) {
+            throw new AssertionError("This method should not be called.");
+        }
+
+        @Override
+        public ReadOnlyAddressBook getAddressBook() {
+            throw new AssertionError("This method should not be called.");
+        }
+
+        @Override
+        public boolean hasPerson(Person person) {
```

Comment:
im not sure if this and addPerson is required, if its required just ignore this comment, if not remove